### ISSUE TITLE
프로젝트 정의서 작성

### ISSUE THREAD

**Issue by ejjang2030** [Dev-FE-3/team4-phoenix-CustomService#2]
# 프로젝트 정의서
[프로젝트 정의서](https://github.com/Dev-FE-3/team4-phoenix-CustomService/blob/main/design/%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EC%A0%95%EC%9D%98%EC%84%9C.md)

## 작업 사항
- 개요 작성
- 시스템 아키텍처 작성
- 설계 상세 작성
    - 사용자 스토리 작성
- 구현 계획 작성 

## 세부 사항
아직 없음

## 논의 사항
아직 없음

## 참고 자료
아직 없음

**Comment by redbuttonking** [comment]
배경 부분이 요구사항 정의서와 중복이 되어서 삭제해도 좋을것 같습니다~!

**Comment by ejjang2030** [comment]
@redbuttonking 네! 수정했습니다! 확인 부탁드려요~! [#24e2c50](https://github.com/Dev-FE-3/team4-phoenix-CustomService/commit/24e2c50d16d25d7ce9e0a65cee6c12bf5d8d0e78)

**Comment by gkfla668** [comment]
사용자 스토리에 예약금과 로그인 관련 내용이 빠져있어요!

**Comment by ejjang2030** [comment]
> 사용자 스토리에 예약금과 로그인 관련 내용이 빠져있어요!

네! 수정할게요~!

**Comment by ejjang2030** [comment]
@gkfla668 [(#b1d9d5f)](https://github.com/Dev-FE-3/team4-phoenix-CustomService/commit/b1d9d5f1f8a20ca70b467ac40702968b70ad0cd8)
사용자 스토리에 예약금과 로그인 관련 내용 추가했습니다~!
이게 맞을지 모르겠네요..ㅎ
